### DISCUSSION TITLE
Understanding Decreased File size of MRC After Unpacking

### DISCUSSION THREAD

**Discussion by braxtonowens** [alexjnoble/jpeg_tomogram#5] [General]
The only reason why the unpacked MRC file would be smaller than the original would be if you are using a smaller data type to represent the pixels. i.e. float16 -> int8. 

Can someone help me understand why an unpacked MRC would be smaller than the original?

**Comment by braxtonowens** [comment]
I guess if you are compressing, gzip might compress an uncompressed jpeg better than the original jpeg.

**Comment by alexjnoble** [comment]
Yes, I think that's right - float16 > int8, which truncates values, but has virtually no affect on features for visualization and picking. And yes, the jpegs could be compressed more - that's a good idea. 

Sorry for not getting to your pull request and issues yet - I've been buried with stuff.

Cheers,
-Alex

**Comment by braxtonowens** [comment]
I did some tests, it does look like gzip can better compress an uncompressed jpeg image. No worries about the PR, take your time, its definitely not the bets PR out there.
